src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calcContainer">
+        <div className="calc">
+          <span className="output">0</span>
+          <div className="horizontalBtn">
+            <div className="verticalBtn">
+              <button type="button" className="btnGray">AC</button>
+              <button type="button" className="btnGray">7</button>
+              <button type="button" className="btnGray">4</button>
+              <button type="button" className="btnGray">1</button>
+            </div>
+            <div className="verticalBtn">
+              <button type="button" className="btnGray">+/-</button>
+              <button type="button" className="btnGray">8</button>
+              <button type="button" className="btnGray">5</button>
+              <button type="button" className="btnGray">2</button>
+            </div>
+            <div className="verticalBtn">
+              <button type="button" className="btnGray">%</button>
+              <button type="button" className="btnGray">9</button>
+              <button type="button" className="btnGray">6</button>
+              <button type="button" className="btnGray">3</button>
+            </div>
+            <div className="verticalBtn">
+              <button type="button" className="btnOrange">÷</button>
+              <button type="button" className="btnOrange">*</button>
+              <button type="button" className="btnOrange">-</button>
+              <button type="button" className="btnOrange">+</button>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly consider your design should match the following given design by adding the last row in the image.
>   ![image](https://user-images.githubusercontent.com/53402600/204617103-545f7cd0-0dd4-4819-becd-492dbbf15944.png)